schemas/schemas/infoAssessment.json

```diff
@@ -92,6 +92,26 @@
             "description": "Requires the student to accept an honor code before starting exam assessments.",
             "type": "boolean",
             "default": true
+        },
+        "groupWork": {
+            "description": "If the assessment will support group work.",
+            "type": "boolean",
+            "default": false
+        },
+        "groupmax": {
```

Comment:
Please use the same naming scheme as other variables in the file (in this case, `camelCase`). Similarly for other variables below.

Also, please use names as similar as possible to the internal database names. So if the database name is `maximum`, let's call this `groupMaximum`, etc.